cfgov/draftail_icons/static/draftail_icons/css/draftail_icons.css

```diff
@@ -0,0 +1,6 @@
+/* Display an img that shows one of our SVG icons with similar styling as cf-icon. */
```

Comment:
Maybe update `cf-icon` to `cf-icon-svg`, since that's the class you're duplicating.

Reply:
Done! [a7b0c4b](https://github.com/cfpb/consumerfinance.gov/pull/8488/commits/a7b0c4bc3defe7b06852329edb4fac0a1c65c7b8)